app/src/main/java/tttTDD/Java/GameConfiguration.java

```diff
@@ -3,12 +3,6 @@
 public class GameConfiguration {
     private int boardSize;
     private boolean isComputer;
-    private int DEFAULT_SIZE = 3;
```

Comment:
Looks deleted, did you have a test that confirmed the default size was 3?

Reply:
yeah I did since it was no longer being used when creating a game. Should the default constructor as well as the size still exists if it's not being called or would you say when the API interface is created, it can be brought back?

Reply:
Cool, nope. You're good. I just missed it.